test/fake-repos/fake-mixin-proto/fake_mixin.proto

```diff
@@ -0,0 +1,8 @@
+// This is a fake proto file for testing.
+
+syntax = "proto3";
+
+package google.fake_mixin
```

Comment:
Is it necessary/useful to put some content into this proto, and have library.proto use it as a mixin?